### DISCUSSION TITLE
How to host my website made with tailwind css to hostinger

### DISCUSSION THREAD

**Discussion by Arvindjitta** [EddieHubCommunity/support#88] [Q&A] [answered]
Hello everyone, does any body know how to host my website made with tailwind css to hostinger

**Comment by eddiejaoude** [comment, marked as answer]
I recommend using GitHub Pages.

Tailwindcss has a few ways to post process / build for hosting https://tailwindcss.com/docs/installation#using-tailwind-cli

- CLI
- Webpack
- Gulp
- *and more on the link above*

The easiest thing to do is use the CDN. It is less performant, but these performance improvements can be made later on. Limitation on using the CDN

- You can't customize Tailwind's default theme
- You can't use any directives like @apply, @variants, etc.
- You can't enable features like group-hover
- You can't install third-party plugins
- You can't tree-shake unused styles
